docs/contents/components/disclosure/accordion/accessibility.ja.mdx

```diff
@@ -0,0 +1,36 @@
+---
+title: Accordion
+description: "`Accordion`は、情報を展開または折りたたんで表示するリストのコンポーネントです。"
+package_name: "@yamada-ui/accordion"
+with_description: true
+tab: アクセシビリティ
+---
+
+`Accordion`は、アクセシビリティに関して[WAI-ARIA - Accordion Pattern](https://www.w3.org/WAI/ARIA/apg/patterns/accordion/)に従います。
+
+## キーボード操作
+
+| キー             | 説明                                                                                                         | 状態                                   |
+| ---------------- | ------------------------------------------------------------------------------------------------------------ | -------------------------------------- |
+| `Tab`            | フォーカスがアコーディオンに移動すると最初の項目に移動し、アコーディオン内である場合、次の項目に移動します。 | -                                      |
+| `Shift` + `Tab`  | フォーカスがアコーディオンに移動すると最後の項目に移動し、アコーディオン内である場合、前の項目に移動します。 | -                                      |
```

Comment:
it is changed from
`フォーカスがアコーディオンに移動すると最後の項目に移動し、アコーディオン内である場合、前の項目に移動します。`
to
`フォーカスがアコーディオン内である場合、前の項目に移動します。`
because, the focus doesn't move to last accordion element by pushing `Shift` + `Tab`